From 8dcfd1cc2dec127f0f16c5c2d0e4704c2aea8927 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Wed, 20 Apr 2022 14:27:57 +0200
Subject: [PATCH 0414/1695] dts: stm32: update IPCC1/2 nodes in stm32mp251.dtsi

Update IPCC1/2 nodes under soc node in stm32mp251.dtsi with its
compatible and clock.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 25bccbac6c7ddcf3b75db1ee932f68f8b28e0739 dts: stm32: update IPCC1/2 nodes in stm32mp251.dtsi]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: I237c205fa8f693fc0c22367aeb1b992326137d53
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/249372
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp251.dtsi | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index 92573e66d..a5ed5d98c 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -116,8 +116,9 @@
 		};
 
 		ipcc1: mailbox@40490000 {
+			compatible = "st,stm32mp25-ipcc";
 			reg = <0x40490000 0x400>;
-			status = "disabled";
+			clocks = <&rcc CK_BUS_IPCC1>;
 		};
 
 		ommanager: ommanager@40500000 {
@@ -716,8 +717,9 @@
 		};
 
 		ipcc2: mailbox@46250000 {
+			compatible = "st,stm32mp25-ipcc";
 			reg = <0x46250000 0x400>;
-			status = "disabled";
+			clocks = <&rcc CK_BUS_IPCC2>;
 		};
 
 		stgenc: stgen@48080000 {
-- 
2.47.2

